### PR TITLE
webtau initial registration

### PR DESCRIPTION
**Pull request by MykolaGolubyev**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    order = "001",
+    id = "001_add_webtau_candidate",
+    author = "MykolaGolubyev"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "webtau",
+      name = "webtau",
+      description =
+        "webtau - web test automation. test across REST-API/Graph QL/Browser/Database/CLI/Business-Logic",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala:18]: This is the description that appears on our website, and also when viewing the `sdk list` command. Can you please make this a little more descriptive and let it be a real sentence/paragraphl?

File: src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      websiteUrl = "https://github.com/testingisdocumenting/webtau"
+    ).insert()
+
+  @ChangeSet(
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala:22]: We don't need this migration as you will be doing your first version release with the API.

**Review by marc0der — CHANGES_REQUESTED** [review]

File: src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala (hunk 1 above)
Comment (MykolaGolubyev) [reply src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala:18]: I copied it as is from the webtau repo. Do you have any suggestions how can I improve it? 
Tool/API allows to write tests that touch one or multiple layers within one test or suite; generate rich report and other goodies. 
What is the size limit?

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala:18]: There is no real size limit, but it is probably best to look at what other vendors are doing for general length and tone.

Comment (MykolaGolubyev) [reply src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala]: updated description

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
Thanks for contributing!

**Pull request merged**